### ISSUE TITLE
about Keyboard Shortcuts

### ISSUE THREAD

**Issue by Raphtalia-chann** [vincelwt/chatgpt-mac#90]
The shortcut (Shift + Command + G) conflicts with Finder's.

**Comment by k-kuraoka** [comment]
I agree.
It is a great app, but I am having trouble with it conflicting with Figma's shortcut keys.
I would like the ability to arbitrarily determine the shortcut key.

**Comment by GeekHank** [comment]
Can anyone tell me what the shortcut is to "send"? (I'm using Mac OS, and the ENTER doesn’t work as sending questions.)

**Comment by tienxu04** [comment]
> Can anyone tell me what the shortcut is to "send"? (I'm using Mac OS, and the ENTER doesn’t work as sending questions.)

yeah im also looking for a way to send message quickly, rather than taking my hands off the keyboard and reach for the mouse

**Comment by johnolek** [comment]
+1 about not including hardcoded keyboard shortcuts. I use Shift + Command + D to send email in Mail and was surprised to find that this is now a global trigger to open this app.

@tienxu04 For quickly sending messages without the mouse, try tab + enter

**Comment by jeffqazgodlike** [comment]
https://user-images.githubusercontent.com/126842705/222621744-4d2af931-3c1f-40bd-97b7-ee1f2664f9a7.mp4
 press enter

**Comment by jeffqazgodlike** [comment]
https://user-images.githubusercontent.com/126842705/222621833-6bc9ddcb-b946-4881-a8ca-6409de890bbd.mp4

press enter

**Comment by tienxu04** [comment]
> +1 about not including hardcoded keyboard shortcuts. I use Shift + Command + D to send email in Mail and was surprised to find that this is now a global trigger to open this app.
> 
> @tienxu04 For quickly sending messages without the mouse, try tab + enter

thanks. been doing that way.

**Comment by tienxu04** [comment]
> press enter

Enter just make a new line, not send.

**Comment by GeekHank** [comment]
> 

Thank you, tab + enter helps a lot!!

**Comment by somefool** [comment]
PLease the option to disable or change the shortcut.
Im never THAT desperate to open the Chat GPT window

**Comment by gmshrek** [comment]
[ChatGPT-Box.dmg](https://github.com/gmshrek/chatgpt-box/releases/download/v1.0.0/ChatGPT-Box.dmg )

It supports custom send shortcut keys: enter or cmd+enter

**Comment by somefool** [comment]
not to send the message, making the app active.

**Comment by codePigPig** [comment]
Looking forward to updating the ability to customize the shortcut keys, thanks

**Comment by gmshrek** [comment]
https://github.com/gmshrek/chatgpt-box
v1.0.1 Have supported. 



| |
gmshrek
|
|
***@***.***
|


---- Replied Message ----
| From | ***@***.***> |
| Date | 03/14/2023 10:06 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [vincelwt/chatgpt-mac] about Keyboard Shortcuts (Issue #90) |

Looking forward to updating the ability to customize the shortcut keys, thanks

**Comment by aaeak** [comment]
> https://github.com/gmshrek/chatgpt-box/releases/tag/v1.0.1 v1.0.1 Have supported.

This is sketchy. Those "Source Code" files have screenshots in them. No Source code, no other work on the account, just a dmg claiming to be the answer you are looking for. And an empty "Have supported." comment submitted by email. Be weary.

**Comment by katzkuma** [comment]
Speaking of sending messages, try using a combination of Command + Enter.
